x/commerciodocs/querier.go

```diff
@@ -50,7 +51,7 @@ func queryGetMetadata(ctx sdk.Context, path []string, keeper Keeper) (res []byte
 
 	bz, err2 := codec.MarshalJSONIndent(keeper.cdc, identityResult)
 	if err2 != nil {
-		panic("Could not marshal result to JSON")
+		return nil, sdk.ErrUnknownRequest(fmt.Sprintf("Could not marshal result to JSON"))
```

Comment:
If we do not have any error variables I think we can remove the usage of `fmt.Sprintf` and leave the play string.